app.js

```diff
@@ -1,28 +1,89 @@
 document.addEventListener("DOMContentLoaded", () => {
-  let currencyData = [
-    { from: "DKK", to: "NGN", rate: 300.31 },
-    { from: "DKK", to: "CAD", rate: 0.19 },
-    { from: "DKK", to: "EUR", rate: 0.13 },
-    { from: "DKK", to: "USD", rate: 0.15 },
-    { from: "DKK", to: "GBP", rate: 0.11 },
-    { from: "USD", to: "DKK", rate: 6.87 },
-    { from: "USD", to: "CAD", rate: 1.31 },
-    { from: "USD", to: "EUR", rate: 0.92 },
-    { from: "USD", to: "GBP", rate: 0.77 },
-    { from: "USD", to: "NGN", rate: 767 },
-    { from: "EUR", to: "DKK", rate: 7.44 },
-    { from: "EUR", to: "CAD", rate: 1.46 },
-    { from: "EUR", to: "GBP", rate: 0.85 },
-    { from: "EUR", to: "NGN", rate: 834 },
-    { from: "GBP", to: "DKK", rate: 8.61 },
-    { from: "GBP", to: "CAD", rate: 1.73 },
-    { from: "GBP", to: "USD", rate: 1.3 },
-    { from: "GBP", to: "NGN", rate: 982.82 },
-    { from: "CAD", to: "DKK", rate: 5.09 },
-    { from: "CAD", to: "USD", rate: 0.76 },
-    { from: "CAD", to: "GBP", rate: 0.57 },
-  ];
+  let currencyData = [];
+
+  function getData() {
+    fetch(
+      "https://raw.githubusercontent.com/oyelajakenny/oyelajakenny.github.io/main/app.json"
+    )
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error(`HTTP error! status: ${response.status}`);
+        }
+        return response.json();
+      })
+      .then((data) => {
+        console.log("Fetched data:", data);
```

Comment:
Avoid leaving debugging code in your project

Reply:
Thank you for the feedback @vdaguenet, duly noted.